src/os/bluestore/BlueStore.cc

```diff
@@ -817,10 +809,7 @@ void BlueStore::TwoQCache::_trim(uint64_t onode_max, uint64_t buffer_max)
       Buffer *b = &*p;
       assert(b->is_clean());
       dout(20) << __func__ << " buffer_warm_in -> out " << *b << dendl;
-      assert(buffer_bytes >= b->length);
-      buffer_bytes -= b->length;
-      assert(buffer_list_bytes[BUFFER_WARM_IN] >= b->length);
-      buffer_list_bytes[BUFFER_WARM_IN] -= b->length;
```

Comment:
We are evicting buffers from the warm_in list here, so the cache_private will be deemed to be BUFFER_WARM_IN. 
Thus the replacement is still equivalent, right?